### DISCUSSION TITLE
HOWTO WebSend SML data (counter reading) every TelePeriod

### DISCUSSION THREAD

**Discussion by hagenbuch** [arendst/Tasmota#16921] [Q&A] [answered]
Hi all,

I have a special compiled tasmota 11.1 0 that is able to read electricity counter data viw SML / OBIS communication via IR interface.

So far, so good.

Now I would like to send two values (production and consumption) via 

`=> WebSend [example.com] /index.php?id=myid&consumption=%consumption%&prodcution=%production% `

to my webserver (using MQTT is not an option here)

.. and I tried to integrate the abovementioned code into the 

>S

or 

>T

partition of the script. Sadly, they are sending almost every second, while TelePeriod should be set to 60 seconds

In short: 

- How do make the script send every 60 seconds?
- How do I get to send values form SML reading?

Cheers
Andreas

**Comment by barbudor** [comment, marked as answer]
Regarding sending at teleperiod, I think you can't sync exactly with Teleperiod message but you can send at the same frequency
At least this is what I understand from a few examples that are using this kind of code:
```
>S
; update display every [`TelePeriod`](Commands#teleperiod)
if upsecs%tper==0
then
=> WebSend [example.com] /index.php?id=myid&consumption=%consumption%&prodcution=%production%
endif
```

And as it won't be in sync, you can even use a different period than Teleperiod with `if upsecs%60==0`


For the other question, I can't help. My knowledge in Scripting language is close to zero

**Reply by hagenbuch** [reply]
Many thanks, sending every minute works now.

I think this part of my question is solved, I still have problems reading the SML values into variables but I will open a different question
